global.json

```diff
@@ -1,5 +1,5 @@
 {
```

Comment:
The global.json changes shouldn't be needed in this review since they were part of the CI pipeline review. Merging from the master branch should remove them from the PR.